js/profile-geonovum.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+// this is only set in a build, not at all in the dev environment
+require.config({
+  paths: {
+    clipboard: "deps/clipboard",
+    hyperhtml: "deps/hyperhtml",
+    idb: "deps/idb",
+    jquery: "deps/jquery",
+    marked: "deps/marked",
+    pluralize: "deps/pluralize",
+    text: "deps/text",
+    webidl2: "deps/webidl2",
+  },
+  shim: {
+    shortcut: {
+      exports: "shortcut",
+    },
+    highlight: {
+      exports: "hljs",
+    },
+  },
+});
+
+define([
+  // order is significant
+  "./core/base-runner",
+  "./core/ui",
+  "./core/reindent",
+  "./core/location-hash",
+  "./core/l10n",
+  "./w3c/defaults",
+  "./core/style",
+  "./w3c/style",
+  "./w3c/l10n",
+  "./core/github",
+  "./core/data-include",
+  "./core/markdown",
+  "./w3c/headers",
+  "./w3c/abstract",
+  "./core/data-transform",
+  "./core/inlines",
+  "./w3c/conformance",
```

Comment:
Same with this one... or other W3C ones below...

Reply:
Agreed